fix: NFT detail screen UI

abdulhaseeb4239 opened the pull request:
# 🔘 PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [x] Bugfix
- [ ] Enhancement
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

# 📜 Background
Small UI differences from design

Issue Link: https://linear.app/xverseapp/issue/ENG-3203/detail-screen-fields-are-not-matching-figma
Context Link (if applicable):

# 🔄 Changes
- added some styling changes 

Impact:
- NFT detail screen UI, no impact on any logic

# 🖼 Screenshot / 📹 Video

<img width="362" alt="Screenshot 2023-11-13 at 4 42 38 PM" src="https://github.com/secretkeylabs/xverse-web-extension/assets/29428363/748e2be2-8ecf-4135-9eb2-e68a2ef93377">


# ✅ Review checklist
Please ensure the following are true before merging:

- [ ] Code Style is consistent with the project guidelines.
- [ ] Code is readable and well-commented.
- [ ] No unnecessary or debugging code has been added.
- [ ] Security considerations have been taken into account.
- [ ] The change has been manually tested and works as expected.
- [ ] Breaking changes and their impacts have been considered and documented.
- [ ] Code does not introduce new technical debt or issues.

teebszet commented:
@abdulhaseeb4239 I didn't load the branch myself, but looking at your screenshot I don't think it looks like this screenshot from linear:
![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/589f7f5a-bd50-41f8-a4d2-0f64c55c5852)

see the column gap is expected to be in the centre, as is the Attributes and the Share/Save button is centre aligned.

abdulhaseeb4239 commented:
> @abdulhaseeb4239 I didn't load the branch myself, but looking at your screenshot I don't think it looks like this screenshot from linear: ![image](https://user-images.githubusercontent.com/6109710/282399581-589f7f5a-bd50-41f8-a4d2-0f64c55c5852.png)
> 
> see the column gap is expected to be in the centre, as is the Attributes and the Share/Save button is centre aligned.

added the fix and updated the screenshot 👍
